checkov/common/bridgecrew/platform_integration.py

```diff
@@ -257,8 +257,10 @@ def setup_bridgecrew_credentials(
                     region_name=region,
                     config=config,
                 )
-                self.platform_integration_configured = True
-                self.use_s3_integration = True
+                if not is_package_scan:
+                    self.skip_fixes = True  # no need to run fixes on CI integration
```

Comment:
Do we need to run fixes on package scanning in CI integration? I don't think so...